src/catalog/catalog-urls.md

```diff
@@ -79,3 +79,23 @@ To prevent search engines from indexing multiple URLs that lead to the same cont
 
    ![]({% link images/images/msg-cache-management.png %}){: .zoom}
    [_Refresh Cache_]({% link system/cache-management.md %})
+
+## Configure catalog media URL format
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand **General** and choose **Web** underneath.
```

Comment:
```suggestion
1. In the left panel, expand **General** and choose **Web**.
```

Reply:
Thank you @shrielenee, for your review!

I have implemented your suggestions, could you please review and approve them?